### ISSUE TITLE
Port 7125 not working (one instance of Klipper)

### ISSUE THREAD

**Issue by VeenstraPieter** [marcolivierarsenault/moonraker-home-assistant#52]
I tried the default port of 7125, but nothing is working.

**Comment by marcolivierarsenault** [comment]
can you start by validating your IP

You should get this available in your browser :
http://192.168.2.30/printer/info

replace with Printer IP or name

**Comment by VeenstraPieter** [comment]
This is what i get:

{"result": {"state": "ready", "state_message": "Printer is ready", "hostname": "klipper", "klipper_path": "/home/klipper/klipper", "python_path": "/home/klipper/klippy-env/bin/python", "log_file": "/home/klipper/printer_data/logs/klippy.log", "config_file": "/home/klipper/printer_data/config/printer.cfg", "software_version": "v0.11.0-128-g57c4da5e", "cpu_info": "2 core Common KVM processor"}}

**Comment by marcolivierarsenault** [comment]
can you validate your port in `moonraker.cfg` 

in my case
```yaml
[server]
host: 0.0.0.0
port: 7125
```

**Comment by VeenstraPieter** [comment]
Yes I allready checked it:

[server]
host: 0.0.0.0
port: 7125
klippy_uds_address: /home/klipper/printer_data/comms/klippy.sock

**Comment by VeenstraPieter** [comment]
oow nooooo, I made a stupid mistake. I'm sorry for wasting your time! Apperently the first time, I did it without the port (simply because I didn't know any better). And the seccond time I researched a little and came to the conclusion it had to be port 7125. But the initial ip was inserted and I didn't change it (it was something like 192.168.1.1xx). After putting in the right ip and the port 7125 it works. Sorry again! Rooky mistake..
